common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -200,6 +200,15 @@ class CapaFields(object):
         scope=Scope.settings
     )
 
+    @property
+    def non_editable_metadata_fields(self):
```

Comment:
There are nine identical overrides to exclude the ``data`` field. Is there any way to have this declared higher up, or in a common mixin or something? It seems brittle to have to spread it around like this.